src/generate/TypesGenerator.spec.ts

```diff
@@ -129,5 +135,60 @@ describe('TypesGenerator', () => {
       void expect(openapiTS).toBeCalledTimes(2);
       void expect(promises.writeFile).toBeCalledTimes(2);
     });
+
+    it('should generate a Typescript file for each spec file and/or dir', async () => {
+      const mockFirstResponse = {
+        data: {
+          tree: [
+            {
+              type: 'blob',
+              path: 'reference',
+              url: 'https://api.github.com/repos/bigcommerce/api-specs/trees/12345',
+            },
+          ],
+        },
+      };
+
+      const mockSecondResponse = {
+        data: {
+          tree: [
+            {
+              type: 'blob',
+              path: 'foo-spec.yaml',
+            },
+            {
+              type: 'blob',
+              path: 'bar-spec.yml',
+            },
+            {
+              type: 'tree',
+              path: 'catalog',
+              url: 'https://api.github.com/repos/bigcommerce/api-specs/trees/123456',
+            },
+          ],
+        },
+      };
+
+      const mockThirdResponse = {
+        data: {
+          tree: [
+            {
+              type: 'blob',
+              path: 'categories-catalog.yml',
+            },
+          ],
+        },
+      };
+
+      mockAxios.get
+        .mockResolvedValueOnce(mockFirstResponse)
+        .mockResolvedValueOnce(mockSecondResponse)
+        .mockResolvedValueOnce(mockThirdResponse);
+
+      void (await tg.generate());
+
+      void expect(openapiTS).toBeCalledTimes(3);
```

Comment:
why do these expects need `void` ? 👀 never seen that before 😅

Reply:
Good catch!  yes it's not necessary 😅  I just copy past it from past test lol, should have checked that.
I'll remove it 👍🏼